File: BarberMe/BarberMe.Database/Models/ShopSettings.cs
namespace BarberMe.Database.Models
{
    public class ShopSettings
    {
        public int ShopSettingsId { get; set; }

        public string Name { get; set; } = null!;

        public string Address { get; set; } = null!;

        public string PhoneNumber { get; set; } = null!;

        public string Email { get; set; } = null!;

        public string? Description { get; set; }
    }
}

File: barberme_mobile/lib/models/shop_settings.dart
class ShopSettings {
  final int id;
  final String name;
  final String address;
  final String phoneNumber;
  final String email;
  final String? description;

  ShopSettings({
    required this.id,
    required this.name,
    required this.address,
    required this.phoneNumber,
    required this.email,
    this.description,
  });

  factory ShopSettings.fromJson(Map<String, dynamic> json) {
    return ShopSettings(
      id: json['id'] as int,
      name: json['name']?.toString() ?? '',
      address: json['address']?.toString() ?? '',
      phoneNumber: json['phoneNumber']?.toString() ?? '',
      email: json['email']?.toString() ?? '',
      description: json['description']?.toString(),
    );
  }
}